### ISSUE TITLE
Bug: WinUI-2.5: Accessibility: [MAS4.2.1]: Narrator: Radio buttons under the "pager display mode" heading are not associated with header.

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#3631]
**User Impact**
If the radio buttons under the "pager display mode" heading are not associated with header, then narrator does not read the header name. Thus it impacts the screen reader user.

**Repro-Steps** 
1. Launch the WinUI_2.5_TestApp
2. Navigate to the Pager control Tab
3. Navigate to the  radio buttons under the "Pager Display Mode" header 
4. Observe the narrator announcement for these radio buttons.

**Expected Behavior**
Radio buttons under the "pager display mode" heading should be associated with header.
The narrator should be say as "pager display mode, number box radio button, 2 of 3, selected" when user traverses the number box radio button.

**Actual Behavior** 
The narrator says as "number box radio button, 2 of 3, selected" when user traverses the number box radio button, but it does not read the associated header name for the radio button.

**Repro-Attachment**
[Narrator-Associated Header with Radio buttons.zip](https://github.com/microsoft/microsoft-ui-xaml/files/5558202/Narrator-Associated.Header.with.Radio.buttons.zip)

**Hardware and build details**
Version Info
Win32
Windows OS Version: Windows 10
App: WinUI_2.5_TestApp

**Comment by Felix-Dev** [comment]
This sounds like a duplicate of https://github.com/microsoft/microsoft-ui-xaml/issues/3183.
